service/src/main/java/uk/gov/hmcts/reform/fpl/enums/SectionType.java

```diff
@@ -0,0 +1,27 @@
+package uk.gov.hmcts.reform.fpl.enums;
+
+public enum SectionType {
+    APPLICANT("mainApplicant", "applicant"),
+    CHILDREN("children", "children"),
+    ORDERS("orders", "orders and directions needed"),
+    GROUNDS("groundsForTheApplication", "grounds for the application"),
+    HEARING("hearing", "hearing needed"),
+    DOCUMENTS("documents", "documents"),
+    CASENAME("caseName", "case name");
+
+    private final String predicate;
+    private final String sectionHeaderName;
+
+    private SectionType(String predicate, String sectionHeaderName) {
+        this.predicate = predicate;
+        this.sectionHeaderName = sectionHeaderName;
+    }
+
+    public String getPredicate() {
```

Comment:
Usually predicate is a boolean-valued function see https://docs.oracle.com/javase/8/docs/api/java/util/function/Predicate.html.

In that content name predicate does not work well for me. What it is is just property name.

Please rename